admin/templates/admin/admin.html

```diff
@@ -81,10 +81,10 @@ <h2 class="card-title">Update Publication Terms</h2>
                             <textarea class="form-control" id="publicationTerms" name="publicationTerms" rows="10"
                                 style="width: 110%;" required>{{ publication_terms|safe }}</textarea>
                         </div>
-                        <button type="submit" class="btn btn-primary">Update Terms</button>
+                        <button type="submit" name="Update Terms" class="btn btn-primary">Update Terms</button>
```

Comment:
Use id instead of name.

The id is to identify the element in the DOM, name is to identify what is posted back to a server.

Reply:
I have used id for UI identification in the scope of this PR. For other UI tests (like maintenance banner), I'll improve with the UI consistency story